romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -228,9 +343,46 @@ def create_image_model(input_model, image_info):
     return im
 
 
-def get_pixeldq_flags(groupdq, pixeldq, slopes, err, gain):
+def determine_diffs2use(ramp_data, integ, row, diffs):
```

Comment:
I wasn't immediately able to see how this PR differed from the stcal equivalent?